tests/fmt.rs

```diff
@@ -1131,6 +1131,12 @@ mod escapes {
             );
         });
     }
+
+    #[test]
+    fn entity_escape_is_not_code_block_indent() {
+        source_range_fmt::assert_events_eq("&#9;foo");
+        source_range_fmt::assert_events_eq("&#32;   foo");
```

Comment:
Also, there is a `stupicat` test which takes a real-world markdown document and runs it through the parsing-serialization pipeline.

There one could add specific sections that would trigger the new code, which makes it easier to understand how this case typically looks like (unit-tests can be abstract).